curriculum/challenges/english/10-coding-interview-prep/the-odin-project/elements-and-tags-question-a.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 637f4e0e72c65bc8e73dfe1e
+videoId: LGQuIIv2RVA
+title: Elements and Tags Question A
+challengeType: 11
+dashedName: elements-and-tags-question-a
+---
+# --description--
+
+Almost all elements on an HTML page are just pieces of content wrapped in opening and closing HTML tags.
+
+Opening tags tell the browser this is the start of an HTML element. They are comprised of a keyword enclosed in angle brackets <>. For example, an opening paragraph tag looks like this: `<p>`.
+
+Closing tags tell the browser where an element ends. They are almost the same as opening tags; the only difference is that they have a forward slash before the keyword. For example, a closing paragraph tag looks like this: `</p>`.
+
+A full paragraph element looks like this:
+
+<img src="https://cdn.statically.io/gh/TheOdinProject/curriculum/90b1a362af0bb8635af9593cd8911c9aefb68569/foundations/html_css/html-foundations/imgs/00.png" alt="element diagram">
```

Comment:
We are relying on an image to present information here - as such, we need descriptive alt text to allow campers who are using assistive technology to receive the same information.